src/DependencyInjection/Configuration.php

```diff
@@ -42,6 +42,18 @@ public function getConfigTreeBuilder(): TreeBuilder
                                         ->booleanNode('infix')->end()
                                         ->booleanNode('optional')->end()
                                         ->booleanNode('sort')->end()
+                                        ->arrayNode('embed')
+                                            ->children()
+                                                ->arrayNode('from')
+                                                    ->scalarPrototype()->end()
+                                                ->end()
+                                                ->arrayNode('model_config')
```

Comment:
It would be great if this node were variable, allowing different configs depending on the embedding adapter used (e.g. https://typesense.org/docs/27.1/api/vector-search.html#using-openai-api), or at least defining all supported options and making them optional

Reply:
You are right, I will thing about it.